docs/intro.md

```diff
@@ -1 +1,38 @@
-**Telescope ALBA** documentation.
+- Assume that a prover has a large collection of data. They want to convince a verifier that their set contains at least a minimum number of elements satisfying a specific condition, known as a predicate. This remains true even if the prover only shares a portion of their data.
+- A trivial solution to this problem would be for the prover to provide the entire dataset or all sequence elements to the verifier. However, this approach is inefficient due to the large size of the data and communication costs.
+- The Approximate Lower Bound Argument (ALBA) protocol is a new cryptographic primitive that solves this problem efficiently.
+- ALBA is designed to prove knowledge efficiently by leveraging an *approximate lower bound*.
+  - This approach establishes a gap between the prover's actual knowledge and the threshold needed to convince the verifier.
+  - The gap enables compact proofs and efficient verification.
+- ALBA supports both centralized and decentralized setups, as well as weighted and unweighted datasets, making it useful for applications such as blockchain protocols, voting systems, and multisignature schemes.
+
+## Overview
+- **The Protocol**
+  - The protocol addresses the challenge of succinctly proving knowledge of a large set of verifiable evidence.
+  - The prover convinces the verifier by revealing only a subset of this evidence, achieving efficiency in both proof size and communication.
+  - Given a large set $S_p$ that satisfies a predicate $R$ such that $|S_p| \geq n_p$, the prover aims to convince the verifier that the set contains more than $n_f$ elements, where $n_f$ is a threshold strictly smaller than $n_p$.
```

Comment:
`cargo doc` doesn't support natively `katex`. We resolve it by adding a custom header. This header should be passed as a flag while compiling the docs.
Also, I cannot locate the flags in the `Cargo.toml` because, the key `build` is not recognized by it. So, I put it in a `config` file. 
I guess you are now able to run the command `cargo doc --open` and render the Latex formulas.

Reply:
Works now, thanks! Could you remove the flag from `Cargo.toml` then?

Reply:
I think the following lines in `Cargo.toml` are still required:
```rust
[package.metadata.docs.rs]
rustdoc-args = [ "--html-in-header", "./docs/assets/katex-header.html" ]
```
since it is not a flag but an argument.

I couldn't compile the documentation either when I removed the below code from `Cargo.toml` or when I moved it to `config`.
I will try to understand its features and let you know.

Reply:
Weird, I'm able to compile docs when I remove this stuff from `Cargo.toml`.